tutorials/setup.md

```diff
@@ -1,3 +1,24 @@
+- [Build and install CRI-O from source](#build-and-install-cri-o-from-source)
```

Comment:
I know it wasn't there before, but could we add the CRI-O logo to the top of this page?
`![CRI-O logo](logo/crio-logo.svg)`

Reply:
oops, the above probably needs to be `../logo/crio-logo.svg`

Reply:
Sure Tom. Looks good now. Just updated the PR.